packages/config/src/index.ts

```diff
@@ -67,3 +67,26 @@ export async function loadConfig<U extends UserConfig>(
 
   return result
 }
+
+const loadConfigCache = new Map<string, LoadConfigResult<UserConfig>>()
+
+export async function createCachedConfigLoader<U extends UserConfig>(
```

Comment:
`createCachedConfigLoader` should be a high-order function that referens a load function, with the cache created inside the function

Reply:
Would you like to make it a closure and put the cache inside the function?
<img width="985" alt="image" src="https://github.com/unocss/unocss/assets/57086651/ad059f3f-2c38-4ca9-8dcf-30bf5fbb01cc">

Reply:
Yeah exactly